exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/SQLServerDialect.kt

```diff
@@ -312,13 +312,7 @@ open class SQLServerDialect : VendorDialect(dialectName, SQLServerDataTypeProvid
         appendIfNotNull(" AUTHORIZATION ", schema.authorization)
     }
 
-    override fun dropSchema(schema: Schema, cascade: Boolean): String = buildString {
-        append("DROP SCHEMA ", schema.identifier)
-
-        if (cascade) {
-            append(" CASCADE")
-        }
-    }
+    override fun dropSchema(schema: Schema, cascade: Boolean): String = "DROP SCHEMA ${schema.identifier}"
```

Comment:
MySQL is the only other dialect to not support CASCADE and it also just [ignores the argument](https://github.com/JetBrains/Exposed/blob/main/exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/Mysql.kt#L382), so for consistency, I've done the same for SQL Server.

An alternative would be to log a warning or throw an `UnsupportedByDialectException`, like some other query builder functions do.

Please let me know if both dialects should be changed to do either.

Reply:
Maybe we can just add a KDoc for this function noting in which dialects CASCADE is not supported? What do you think?

Reply:
Since it's only 2, I can definitely do that. It's not the most visible, as I'm noticing by going through issues and questions that KDocs aren't always looked at by users.

Making the KDocs the only visible warning (other than looking at source code directly) only becomes less ideal if we start increasing the databases that Exposed supports. Because we'd have to remember to look at all relevant KDocs to potentially edit them every time (and that's not taking into account different database versions). But that's only something to consider if we make it a habit to include a list of unsupported DB for every SQL-generating function in all KDocs. 

The dream would be to have a (website) documentation page for every single query function that lists an example of the generated SQL in each database dialect. So users can see what they'll get without having to try/log/fail/repeat.

Reply:
Let's make the KDoc anyway. We also can make an overload for cascade parameter and mark it as `@Deprecated` with a level warning